scriptmodules/supplementary/wifi.sh

```diff
@@ -214,6 +218,8 @@ The file should contain two lines as follows\n\nssid = \"YOUR WIFI SSID\"\npsk =
                     connect_wifi
                     ;;
                 2)
+                    dialog --defaultno --yesno "This will remove the WiFi configuration and stop the WiFi.\n\nAre you sure you want to continue ?" 12 35
```

Comment:
Sorry this needs the redirection also.

Reply:
Should be ok now. Ideally I should have split the extra changes into a separate commit. If anymore issues come up, I can do it so it's easier to inspect.